pages/public-sitemap.tsx

```diff
@@ -120,7 +121,9 @@ export async function getStaticProps() {
         sitemap.docs[category] ??= [];
         sitemap.docs[category].push({
           name: titleize(
-            rest.length > 1 ? rest.slice(-1)[0] : rest[0] ?? category
+            rest.length > 1
+              ? `${rest.slice(-2)[0]}:  ${rest.slice(-2)[1]}`
+              : rest[0] ?? category
```

Comment:
It might be helpful to throw this into its own function so it's more clear what's going on.